File: Server/Authentication.php
<?php
/**
 * Created by PhpStorm.
 * User: ezhor
 * Date: 23/01/2018
 * Time: 19:58
 */

class Authentication
{
    private $id;
    private $usuario;
    private $contrasena;
    private $token;

    /**
     * Authentication constructor.
     * @param $id
     * @param $usuario
     * @param $contrasena
     * @param $token
     */
    public function __construct($id, $usuario, $contrasena, $token)
    {
        $this->id = $id;
        $this->usuario = $usuario;
        $this->contrasena = $contrasena;
        $this->token = $token;
    }

    /**
     * @return mixed
     */
    public function getId()
    {
        return $this->id;
    }

    /**
     * @param mixed $id
     */
    public function setId($id)
    {
        $this->id = $id;
    }




    /**
     * @return mixed
     */
    public function getUsuario()
    {
        return $this->usuario;
    }

    /**
     * @param mixed $usuario
     */
    public function setUsuario($usuario)
    {
        $this->usuario = $usuario;
    }

    /**
     * @return mixed
     */
    public function getContrasena()
    {
        return $this->contrasena;
    }

    /**
     * @param mixed $contrasena
     */
    public function setContrasena($contrasena)
    {
        $this->contrasena = $contrasena;
    }

    /**
     * @return mixed
     */
    public function getToken()
    {
        return $this->token;
    }

    /**
     * @param mixed $token
     */
    public function setToken($token)
    {
        $this->token = $token;
    }


}

File: GrumpyWorld/app/src/main/java/com/arensis_games/grumpyworld/conection/Authentication.java
package com.arensis_games.grumpyworld.conection;

/**
 * Created by dparrado on 31/01/18.
 */

public class Authentication {
    private String usuario;
    private String contrasena;

    public Authentication() {
    }

    public Authentication(String usuario, String contrasena) {
        this.usuario = usuario;
        this.contrasena = contrasena;
    }

    public String getUsuario() {
        return usuario;
    }

    public void setUsuario(String usuario) {
        this.usuario = usuario;
    }

    public String getContrasena() {
        return contrasena;
    }

    public void setContrasena(String contrasena) {
        this.contrasena = contrasena;
    }
}
